docs/manual.md

```diff
@@ -616,6 +616,29 @@ and `$ignore_marker end`.  This syntax is also useful for ledger
 	include ledger-specific-header.ledger
 	; NoL2B end
 
+Since some people use ledger and beancount in parallel using
+ledger2beancount, it is sometimes useful to put beancount-specific
+commands in the input file.  Of course, they may not be valid in ledger.
+Therefore, you can put a commented out line in the ledger input, mark it
+with the `$keep_marker` and ledger2beancount will uncomment the line and
+put it in the output.
+
+Given the input
+
+	; 2013-11-03 note Liabilities:CreditCard "Called about fraud" ; L2Bonly
+
```

Comment:
we should either add "L2Bonly" as the default value for $keep_marker (my preference), or add here another documentation line like:
```
and the configuration value:

  keep_marker: L2Bonly
```

Reply:
Yeah, I was wondering about that.